pkgs/applications/graphics/shutter/default.nix

```diff
@@ -29,6 +31,7 @@ stdenv.mkDerivation rec {
     wrapProgram $out/bin/shutter \
       --set PERL5LIB "${stdenv.lib.makePerlPath perlModules}" \
       --prefix PATH : "${imagemagick.out}/bin" \
+      --prefix XDG_DATA_DIRS : "${hicolor-icon-theme}/share" \
```

Comment:
Putting `hicolor-icon-theme` first means it will have precedence over any user-installed themes in `XDG_DATA_DIRS`.
If it is meant as a fallback for missing icons, `--suffix` might be better here.

Reply:
you're absolutely right, missed that, thanks!